spec/arrays.dd

```diff
@@ -418,12 +418,12 @@ assert(s == [2, 3, 3, 4]);
 )
 
 
-$(H2 $(LNAME2 array-setting, Array Setting))
+$(H2 $(LNAME2 array-setting, Array Filling))
 
         $(P If a slice operator appears as the left-hand side of an assignment
         expression, and the type of the right-hand side is the same as the
-        element type of the left-hand side, then the array contents of the
-        left-hand side are set to the right-hand side.
+        element type of the left-hand side, then each element of the slice
+        expression will be assigned to the value on the right-hand side.
```

Comment:
BTW, 'slice expression' is much easier to talk about than a `SliceOperation` (https://github.com/dlang/dlang.org/pull/3680), as the latter is awkward and not an expression. I prefer it before when we had `SliceExpression`, which is more natural and consistent with the other docs.